Ladybird/Android/build.gradle.kts

```diff
@@ -89,4 +91,5 @@ dependencies {
     androidTestImplementation("androidx.test.ext:junit:1.1.5")
     androidTestImplementation("androidx.test.ext:junit-ktx:1.1.5")
     androidTestImplementation("androidx.test.espresso:espresso-core:3.5.1")
+    implementation("com.google.oboe:oboe:1.9.0")
```

Comment:
For my understanding, does this mean that we're asking gradle to make the oboe library available to our CMake targets? It's fine to not use vcpkg if there's a more Android-y way to retrieve packages.

Reply:
Yeah, it's the official way to use this library